scripts/install.sh

```diff
@@ -34,13 +34,17 @@ _detect_os(){
 _download_url() {
         local arch="$(_detect_arch)"
         local os="$(_detect_os)"
+        local version=""
```

Comment:
you can set this to `local version = "$KGW_VERSION"` and then remove the `else ... fi` clause from below